extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -1,21 +1,12 @@
 import {CSS} from '../../../build/amp-fit-text-0.1.css';
-import {
-  applyFillContent,
-  getLengthNumeral,
-  isLayoutSizeDefined,
-} from '#core/dom/layout';
+import {getLengthNumeral, isLayoutSizeDefined} from '#core/dom/layout';
```

Comment:
Since this PR seems touches this extension, consider removing it from `extensions/import-order-excluded.json`

Reply:
Happy to followup with that.